# Translations template for sahara.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the sahara project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2014.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: sahara 2015.1.dev22.g398fa34\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-10-04 06:10+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"

#: sahara/config.py:94
#, python-format
msgid "Option '%(option)s' is required for config group '%(group)s'"
msgstr ""

#: sahara/config.py:102
msgid ""
"use_namespaces can not be set to \"True\" when use_neutron is set to "
"\"False\""
msgstr ""

#: sahara/context.py:112
msgid "Context isn't available here"
msgstr ""

#: sahara/exceptions.py:27
msgid "An unknown exception occurred"
msgstr ""

#: sahara/exceptions.py:39
#, python-format
msgid "Object '%s' is not found"
msgstr ""

#: sahara/exceptions.py:53
msgid "Name already exists"
msgstr ""

#: sahara/exceptions.py:62
msgid "Invalid credentials"
msgstr ""

#: sahara/exceptions.py:71
msgid "Invalid object reference"
msgstr ""

#: sahara/exceptions.py:80
#, python-format
msgid "Error during command execution: \"%s\""
msgstr ""

#: sahara/exceptions.py:112
msgid "Data is invalid"
msgstr ""

#: sahara/exceptions.py:121
msgid "Job binary internal data must be a string of length greater than zero"
msgstr ""

#: sahara/exceptions.py:131
msgid ""
"To work with JobBinary located in internal swift add 'user' and "
"'password' to extra"
msgstr ""

#: sahara/exceptions.py:141
msgid "Database object already exists"
msgstr ""

#: sahara/exceptions.py:150
msgid "Object was not created"
msgstr ""

#: sahara/exceptions.py:159
msgid "Object was not deleted"
msgstr ""

#: sahara/exceptions.py:169
#, python-format
msgid "Node Group %s is missing 'floating_ip_pool' field"
msgstr ""

#: sahara/exceptions.py:187
#, python-format
msgid "Size of data (%(size)s) is greater than maximum (%(maximum)s)"
msgstr ""

#: sahara/exceptions.py:199
#, python-format
msgid "An error occurred in thread '%(thread)s': %(e)s"
msgstr ""

#: sahara/exceptions.py:209
#, python-format
msgid "Feature '%s' is not implemented"
msgstr ""

#: sahara/exceptions.py:215
#, python-format
msgid "Heat stack failed with status %s"
msgstr ""

#: sahara/exceptions.py:249
#, python-format
msgid "Operation timed out after %i second(s)"
msgstr ""

#: sahara/api/base.py:22
msgid "This API operation isn't implemented"
msgstr ""

#: sahara/api/middleware/auth_valid.py:55
msgid "Incorrect path"
msgstr ""

#: sahara/api/middleware/auth_valid.py:61
msgid "Token tenant != requested tenant"
msgstr ""

#: sahara/conductor/__init__.py:32
msgid "Remote conductor isn't implemented yet."
msgstr ""

#: sahara/conductor/resource.py:118
#, python-format
msgid "Unsupported type: %s"
msgstr ""

#: sahara/db/migration/cli.py:41
msgid "You must provide a revision or relative delta"
msgstr ""

#: sahara/db/sqlalchemy/api.py:192
#, python-format
msgid "Duplicate entry for Cluster: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:202
#, python-format
msgid "Duplicate entry for NodeGroup: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:214 sahara/db/sqlalchemy/api.py:226
#: sahara/db/sqlalchemy/api.py:245
#, python-format
msgid "Cluster id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:261 sahara/db/sqlalchemy/api.py:273
#: sahara/db/sqlalchemy/api.py:292
#, python-format
msgid "Node Group id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:311 sahara/db/sqlalchemy/api.py:322
#: sahara/db/sqlalchemy/api.py:339 sahara/db/sqlalchemy/api.py:350
#, python-format
msgid "Instance id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:383
#, python-format
msgid "Duplicate entry for ClusterTemplate: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:394
#, python-format
msgid "Duplicate entry for TemplatesRelation: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:407
#, python-format
msgid "Cluster Template id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:439
#, python-format
msgid "Duplicate entry for NodeGroupTemplate: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:452
#, python-format
msgid "Node Group Template id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:483
#, python-format
msgid "Duplicate entry for DataSource: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:496
#, python-format
msgid "Data Source id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:500 sahara/db/sqlalchemy/api.py:637
msgid " on foreign key constraint"
msgstr ""

#: sahara/db/sqlalchemy/api.py:501
#, python-format
msgid "Data Source deletion failed%s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:535
#, python-format
msgid "Duplicate entry for JobExecution: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:547 sahara/db/sqlalchemy/api.py:559
#, python-format
msgid "JobExecution id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:608
#, python-format
msgid "Duplicate entry for Job: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:620 sahara/db/sqlalchemy/api.py:633
#, python-format
msgid "Job id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:638
#, python-format
msgid "Job deletion failed%s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:679
#, python-format
msgid "Duplicate entry for JobBinary: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:700
#, python-format
msgid "JobBinary id '%s' not found!"
msgstr ""

#: sahara/db/sqlalchemy/api.py:704
msgid "JobBinary is referenced and cannot be deleted"
msgstr ""

#: sahara/db/sqlalchemy/api.py:744 sahara/db/sqlalchemy/api.py:762
#, python-format
msgid ""
"Size of internal binary (%(size)sKB) is greater than the maximum "
"(%(maximum)sKB)"
msgstr ""

#: sahara/db/sqlalchemy/api.py:774
#, python-format
msgid "Duplicate entry for JobBinaryInternal: %s"
msgstr ""

#: sahara/db/sqlalchemy/api.py:787
#, python-format
msgid "JobBinaryInternal id '%s' not found!"
msgstr ""

#: sahara/openstack/common/exception.py:103
msgid "Uncaught exception"
msgstr ""

#: sahara/openstack/common/gettextutils.py:301
msgid "Message objects do not support addition."
msgstr ""

#: sahara/openstack/common/gettextutils.py:311
msgid ""
"Message objects do not support str() because they may contain non-ascii "
"characters. Please use unicode() or translate() instead."
msgstr ""

#: sahara/openstack/common/lockutils.py:101
#, python-format
msgid "Unable to acquire lock on `%(filename)s` due to %(exception)s"
msgstr ""

#: sahara/openstack/common/log.py:290
#, python-format
msgid "Deprecated: %s"
msgstr ""

#: sahara/openstack/common/log.py:398
#, python-format
msgid "Error loading logging config %(log_config)s: %(err_msg)s"
msgstr ""

#: sahara/openstack/common/log.py:459
#, python-format
msgid "syslog facility must be one of: %s"
msgstr ""

#: sahara/openstack/common/log.py:710
#, python-format
msgid "Fatal call to deprecated config: %(msg)s"
msgstr ""

#: sahara/openstack/common/periodic_task.py:40
#, python-format
msgid "Unexpected argument for periodic task creation: %(arg)s."
msgstr ""

#: sahara/openstack/common/processutils.py:59
msgid "Unexpected error while running command."
msgstr ""

#: sahara/openstack/common/processutils.py:62
#, python-format
msgid ""
"%(description)s\n"
"Command: %(cmd)s\n"
"Exit code: %(exit_code)s\n"
"Stdout: %(stdout)r\n"
"Stderr: %(stderr)r"
msgstr ""

#: sahara/openstack/common/processutils.py:143
#, python-format
msgid "Got unknown keyword args: %r"
msgstr ""

#: sahara/openstack/common/processutils.py:148
msgid "Command requested root, but did not specify a root helper."
msgstr ""

#: sahara/openstack/common/processutils.py:158
#, python-format
msgid "Running cmd (subprocess): %s"
msgstr ""

#: sahara/openstack/common/processutils.py:206
#, python-format
msgid "%r failed. Retrying."
msgstr ""

#: sahara/openstack/common/processutils.py:248
msgid "Environment not supported over SSH"
msgstr ""

#: sahara/openstack/common/processutils.py:252
msgid "process_input not supported over SSH"
msgstr ""

#: sahara/openstack/common/strutils.py:125
#, python-format
msgid "Unrecognized value '%(val)s', acceptable values are: %(acceptable)s"
msgstr ""

#: sahara/openstack/common/strutils.py:230
#, python-format
msgid "Invalid unit system: \"%s\""
msgstr ""

#: sahara/openstack/common/strutils.py:239
#, python-format
msgid "Invalid string format: %s"
msgstr ""

#: sahara/openstack/common/wsgi.py:187
msgid "cannot understand JSON"
msgstr ""

#: sahara/openstack/common/wsgi.py:211
msgid "cannot understand XML"
msgstr ""

#: sahara/plugins/base.py:103
#, python-format
msgid "Plugin with name '%s' already exists."
msgstr ""

#: sahara/plugins/base.py:114
#, python-format
msgid "Plugins couldn't be loaded: %s"
msgstr ""

#: sahara/plugins/provisioning.py:110
#, python-format
msgid "Can't find applicable target '%(applicable_target)s' for '%(config_name)s'"
msgstr ""

#: sahara/plugins/provisioning.py:117
#, python-format
msgid "Can't find config '%(config_name)s' in '%(applicable_target)s'"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:88
msgid "'cluster' or 'instance' argument missed"
msgstr ""

#: sahara/plugins/cdh/cloudera_utils.py:98
#, python-format
msgid "Process %(process)s is not supported by CDH plugin"
msgstr ""

#: sahara/plugins/cdh/commands.py:68
msgid "OS on image is not supported by CDH plugin"
msgstr ""

#: sahara/plugins/cdh/deploy.py:266
msgid "Cloudera agents failed to connect to Cloudera Manager"
msgstr ""

#: sahara/plugins/cdh/deploy.py:295
#, python-format
msgid ""
"Cloudera Manager failed to start in %(timeout)s minutes on node "
"'%(node)s' of cluster '%(cluster)s'"
msgstr ""

#: sahara/plugins/cdh/plugin.py:35
msgid ""
"This plugin provides an ability to launch CDH clusters with Cloudera "
"Manager management console."
msgstr ""

#: sahara/plugins/cdh/validation.py:32
msgid "'cm_api' is not installed."
msgstr ""

#: sahara/plugins/cdh/validation.py:94 sahara/plugins/cdh/validation.py:114
#, python-format
msgid "CDH plugin cannot scale nodegroup with processes: %(processes)s"
msgstr ""

#: sahara/plugins/cdh/validation.py:100
msgid ""
"CDH plugin cannot scale node group with processes which have no master-"
"processes run in cluster"
msgstr ""

#: sahara/plugins/fake/plugin.py:26
msgid ""
"It's a fake plugin that aimed to work on the CirrOS images. It doesn't "
"install Hadoop. It's needed to be able to test provisioning part of "
"Sahara codebase itself."
msgstr ""

#: sahara/plugins/general/exceptions.py:22
#, python-format
msgid "Chosen node group %(ng_name)s cannot be scaled : %(reason)s"
msgstr ""

#: sahara/plugins/general/exceptions.py:29
msgid "Failed to decommission cluster"
msgstr ""

#: sahara/plugins/general/exceptions.py:38
#, python-format
msgid "Cluster %(cluster_name)s cannot be scaled : %(reason)s"
msgstr ""

#: sahara/plugins/general/exceptions.py:48
#, python-format
msgid "Cluster is missing a service: %s"
msgstr ""

#: sahara/plugins/general/exceptions.py:51
#, python-format
msgid "%(message)s, required by service: %(required_by)s"
msgstr ""

#: sahara/plugins/general/exceptions.py:69
#, python-format
msgid ""
"Hadoop cluster should contain %(expected_count)s %(component)s "
"component(s). Actual %(component)s count is %(count)s"
msgstr ""

#: sahara/plugins/general/exceptions.py:91
#, python-format
msgid "Failed to Provision Hadoop Cluster: %s"
msgstr ""

#: sahara/plugins/general/utils.py:42
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:68
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:75
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:97
msgid "0 or 1"
msgstr ""

#: sahara/plugins/hdp/ambariplugin.py:207
msgid "An Ambari user in the admin group must be configured."
msgstr ""

#: sahara/plugins/hdp/ambariplugin.py:262
msgid ""
"The Hortonworks OpenStack plugin works with project Sahara to automate "
"the deployment of the Hortonworks data platform on OpenStack based public"
" & private clouds"
msgstr ""

#: sahara/plugins/hdp/clusterspec.py:317
#, python-format
msgid ""
"Could not determine property type for property '%(property)s' with value:"
" %(value)s"
msgstr ""

#: sahara/plugins/hdp/configprovider.py:65
#, python-format
msgid "Internal Error. Duplicate property name detected: %s"
msgstr ""

#: sahara/plugins/hdp/hadoopserver.py:86
msgid "Failed to install Hortonworks Ambari"
msgstr ""

#: sahara/plugins/hdp/hadoopserver.py:111
msgid "Failed to install Hadoop Swift integration"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/services.py:523
#: sahara/plugins/hdp/versions/version_2_0_6/services.py:576
#, python-format
msgid "Invalid value for property 'hbase-site/hbase.rootdir' : %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:87
#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:601
msgid ""
"The HDP plugin does not support the decommissioning of nodes for HDP "
"version 1.3.2"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:188
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:162
#, python-format
msgid "Failed to add cluster: %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:237
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:211
#, python-format
msgid "Failed to set configurations on cluster: %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:253
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:228
#, python-format
msgid "Failed to add services to cluster: %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:271
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:247
#, python-format
msgid "Failed to add components to services: %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:289
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:265
#, python-format
msgid "Failed to add host: %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:305
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:284
#, python-format
msgid "Failed to add host component: %s"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:332
#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:337
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:311
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:316
msgid "Installation of Hadoop stack failed."
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:378
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:363
msgid "Unable to finalize Ambari state."
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:409
#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:417
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:393
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:401
msgid "Start of Hadoop services failed."
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:436
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:420
msgid "Failed to change state of Hadoop components"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:443
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:427
msgid "Hadoop/Ambari command failed."
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:510
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:494
msgid ""
"Unable to determine installed service components in scaled instances.  "
"status code returned = {0}"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:550
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:534
msgid "Unable to update Ambari admin user credentials: {0}"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:566
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:550
msgid "Unable to create Ambari user: {0}"
msgstr ""

#: sahara/plugins/hdp/versions/version_1_3_2/versionhandler.py:576
#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:560
#, python-format
msgid "Unable to delete Ambari user: %(user_name)s : %(text)s"
msgstr ""

#: sahara/plugins/hdp/versions/version_2_0_6/versionhandler.py:632
msgid ""
"An error occurred while trying to decommission the DataNode instances "
"that are being shut down. Please consult the Ambari server logs on the "
"master node for more information about the failure."
msgstr ""

#: sahara/plugins/spark/config_helper.py:207
#, python-format
msgid "Unable to get parameter '%(param_name)s' from service %(service)s"
msgstr ""

#: sahara/plugins/spark/edp_engine.py:25
#, python-format
msgid "Spark 1.0.0 or higher required to run spark %s jobs"
msgstr ""

#: sahara/plugins/spark/plugin.py:54
msgid ""
"This plugin provides an ability to launch Spark on Hadoop CDH cluster "
"without any management consoles."
msgstr ""

#: sahara/plugins/spark/plugin.py:75 sahara/plugins/spark/plugin.py:90
msgid "1 or more"
msgstr ""

#: sahara/plugins/spark/plugin.py:390 sahara/plugins/spark/plugin.py:404
#, python-format
msgid "Spark plugin cannot scale nodegroup with processes: %s"
msgstr ""

#: sahara/plugins/spark/plugin.py:414
#, python-format
msgid ""
"Spark plugin cannot shrink cluster because there would be not enough "
"nodes for HDFS replicas (replication factor is %s)"
msgstr ""

#: sahara/plugins/spark/scaling.py:92
#: sahara/plugins/vanilla/hadoop2/scaling.py:132
#: sahara/plugins/vanilla/v1_2_1/scaling.py:79
#, python-format
msgid "Cannot finish decommission of cluster %(cluster)s in %(seconds)d seconds"
msgstr ""

#: sahara/plugins/vanilla/plugin.py:27
msgid ""
"This plugin provides an ability to launch vanilla Apache Hadoop cluster "
"without any management consoles. Also it can deploy Oozie and Hive"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/config_helper.py:161
#, python-format
msgid "Unable to get parameter '%(name)s' from service %(service)s"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/run_scripts.py:55
#, python-format
msgid "Process %s is not supported"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/validation.py:71
msgid "Number of datanodes must be not less than dfs.replication."
msgstr ""

#: sahara/plugins/vanilla/hadoop2/validation.py:83
#: sahara/plugins/vanilla/hadoop2/validation.py:102
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:448
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:467
#, python-format
msgid "Vanilla plugin cannot scale nodegroup with processes: %s"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/validation.py:88
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:453
msgid ""
"Vanilla plugin cannot scale node group with processes which have no "
"master-processes run in cluster"
msgstr ""

#: sahara/plugins/vanilla/hadoop2/validation.py:111
#: sahara/plugins/vanilla/v1_2_1/versionhandler.py:477
#, python-format
msgid ""
"Vanilla plugin cannot shrink cluster because it would be not enough nodes"
" for replicas (replication factor is %s)"
msgstr ""

#: sahara/plugins/vanilla/v1_2_1/config_helper.py:213
#, python-format
msgid "Unable get parameter '%(parameter)s' from service %(service)s"
msgstr ""

#: sahara/plugins/vanilla/v2_3_0/versionhandler.py:60
msgid ""
"The Vanilla 2.3.0 plugin is now deprecated and will be removed in the "
"Kylo release. The Vanilla 2.4.1 plugin remains and continues to be "
"supported."
msgstr ""

#: sahara/service/direct_engine.py:176
#, python-format
msgid "Server group with name %s is already exists"
msgstr ""

#: sahara/service/direct_engine.py:199
#, python-format
msgid "Several server groups with name %s found"
msgstr ""

#: sahara/service/direct_engine.py:397
#, python-format
msgid "Node %s has error status"
msgstr ""

#: sahara/service/trusts.py:65
msgid "Failed to create trust"
msgstr ""

#: sahara/service/trusts.py:103
msgid "Failed to delete trust {0}"
msgstr ""

#: sahara/service/validation.py:77
#, python-format
msgid "Object with %s not found"
msgstr ""

#: sahara/service/volumes.py:62
#, python-format
msgid "Error attach volume to instance %s"
msgstr ""

#: sahara/service/volumes.py:91
#, python-format
msgid "Volume %s has error status"
msgstr ""

#: sahara/service/edp/job_manager.py:92
#, python-format
msgid "Cluster does not support job type %s"
msgstr ""

#: sahara/service/edp/binary_retrievers/internal_swift.py:67
#, python-format
msgid "Url for binary in internal swift must start with %s"
msgstr ""

#: sahara/service/edp/binary_retrievers/internal_swift.py:85
#, python-format
msgid ""
"Size of swift container (%(size)sKB) is greater than maximum "
"(%(maximum)sKB)"
msgstr ""

#: sahara/service/edp/binary_retrievers/internal_swift.py:109
#, python-format
msgid "Size of swift object (%(size)sKB) is greater than maximum (%(maximum)sKB)"
msgstr ""

#: sahara/service/edp/oozie/workflow_creator/base_workflow.py:49
#, python-format
msgid "\"%s\" child cannot be added to prepare element"
msgstr ""

#: sahara/service/edp/oozie/workflow_creator/base_workflow.py:60
#, python-format
msgid "\"%s\" child cannot be added to streaming element"
msgstr ""

#: sahara/service/edp/spark/engine.py:190
#, python-format
msgid "Spark job execution failed. Exit status = %(status)s, stdout = %(stdout)s"
msgstr ""

#: sahara/service/validations/base.py:55
#, python-format
msgid "Sahara doesn't contain plugin with name '%s'"
msgstr ""

#: sahara/service/validations/base.py:61
#, python-format
msgid "Requested plugin '%(name)s' doesn't support version '%(version)s'"
msgstr ""

#: sahara/service/validations/base.py:68
#, python-format
msgid "Requested image '%s' is not registered"
msgstr ""

#: sahara/service/validations/base.py:79
#, python-format
msgid "Plugin doesn't contain applicable target '%s'"
msgstr ""

#: sahara/service/validations/base.py:84
#, python-format
msgid ""
"Plugin's applicable target '%(target)s' doesn't contain config with name "
"'%(name)s'"
msgstr ""

#: sahara/service/validations/base.py:148
#, python-format
msgid "Requested flavor '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:157
#, python-format
msgid "Security group '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:172
#, python-format
msgid "Floating IP pool %(pool)s for node group '%(group)s' not found"
msgstr ""

#: sahara/service/validations/base.py:179
msgid "Duplicates in node processes have been detected"
msgstr ""

#: sahara/service/validations/base.py:187
#, python-format
msgid "Plugin supports the following node procesess: %s"
msgstr ""

#: sahara/service/validations/base.py:195
msgid "Duplicates in node group names are detected"
msgstr ""

#: sahara/service/validations/base.py:205
#, python-format
msgid "Security group with name '%s' already exists"
msgstr ""

#: sahara/service/validations/base.py:212
#, python-format
msgid "Availability zone '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:220
#, python-format
msgid "Cluster with name '%s' already exists"
msgstr ""

#: sahara/service/validations/base.py:229
#, python-format
msgid "Cluster name '%s' is already used as Heat stack name"
msgstr ""

#: sahara/service/validations/base.py:241
#, python-format
msgid ""
"Composite hostname %(host)s in provisioned cluster exceeds maximum limit "
"%(limit)s characters"
msgstr ""

#: sahara/service/validations/base.py:252
#, python-format
msgid "Requested keypair '%s' not found"
msgstr ""

#: sahara/service/validations/base.py:257
#, python-format
msgid "Network %s not found"
msgstr ""

#: sahara/service/validations/base.py:265
#, python-format
msgid "Cluster template with name '%s' already exists"
msgstr ""

#: sahara/service/validations/base.py:271
#, python-format
msgid "Cluster template with id '%s' doesn't exist"
msgstr ""

#: sahara/service/validations/base.py:292
#, python-format
msgid "NodeGroup template with name '%s' already exists"
msgstr ""

#: sahara/service/validations/base.py:298
#, python-format
msgid "NodeGroup template with id '%s' doesn't exist"
msgstr ""

#: sahara/service/validations/base.py:333
#, python-format
msgid "Cluster doesn't contain node group with name '%s'"
msgstr ""

#: sahara/service/validations/base.py:347
#, python-format
msgid "Can't add new nodegroup. Cluster already has nodegroup with name '%s'"
msgstr ""

#: sahara/service/validations/base.py:361
msgid "Cinder is not supported"
msgstr ""

#: sahara/service/validations/base.py:373
#, python-format
msgid ""
"Tags of requested image '%(image)s' don't contain required tags "
"['%(name)s', '%(version)s']"
msgstr ""

#: sahara/service/validations/cluster_templates.py:137
#, python-format
msgid "Cluster template %(id)s in use by %(clusters)s"
msgstr ""

#: sahara/service/validations/clusters.py:75
#: sahara/service/validations/clusters.py:97
#, python-format
msgid "'%s' field is not found"
msgstr ""

#: sahara/service/validations/clusters.py:91
msgid "'neutron_management_network' field can't be used with 'use_neutron=False'"
msgstr ""

#: sahara/service/validations/clusters_scaling.py:80
#, python-format
msgid "Cluster created before Juno release can't be scaled with %(engine)s engine"
msgstr ""

#: sahara/service/validations/clusters_scaling.py:87
#, python-format
msgid ""
"Cluster created with %(old_engine)s infrastructure engine can't be scaled"
" with %(new_engine)s engine"
msgstr ""

#: sahara/service/validations/clusters_scaling.py:97
#, python-format
msgid "Requested plugin '%s' doesn't support cluster scaling feature"
msgstr ""

#: sahara/service/validations/clusters_scaling.py:102
#, python-format
msgid "Cluster cannot be scaled not in 'Active' status. Cluster status: %s"
msgstr ""

#: sahara/service/validations/node_group_templates.py:123
#, python-format
msgid ""
"Node group template %(template)s is in use by cluster templates: "
"%(users)s; and clusters: %(clusters)s"
msgstr ""

#: sahara/service/validations/plugins.py:27
#, python-format
msgid ""
"Requested plugin '%s' doesn't support converting config files to cluster "
"templates"
msgstr ""

#: sahara/service/validations/edp/base.py:53
#, python-format
msgid "Data source with name '%s' already exists"
msgstr ""

#: sahara/service/validations/edp/base.py:59
#, python-format
msgid "DataSource with id '%s' doesn't exist"
msgstr ""

#: sahara/service/validations/edp/base.py:65
#, python-format
msgid "Job with name '%s' already exists"
msgstr ""

#: sahara/service/validations/edp/base.py:71
#, python-format
msgid "JobBinaryInternal with id '%s' doesn't exist"
msgstr ""

#: sahara/service/validations/edp/base.py:80
#, python-format
msgid "Provided input and output DataSources reference the same location: %s"
msgstr ""

#: sahara/service/validations/edp/data_source.py:67
msgid "Swift url must not be empty"
msgstr ""

#: sahara/service/validations/edp/data_source.py:70
msgid "URL scheme must be 'swift'"
msgstr ""

#: sahara/service/validations/edp/data_source.py:79
#, python-format
msgid "URL must be of the form swift://container%s/object"
msgstr ""

#: sahara/service/validations/edp/data_source.py:83
msgid "No credentials provided for Swift"
msgstr ""

#: sahara/service/validations/edp/data_source.py:87
msgid "User is not provided in credentials for Swift"
msgstr ""

#: sahara/service/validations/edp/data_source.py:91
msgid "Password is not provided in credentials for Swift"
msgstr ""

#: sahara/service/validations/edp/data_source.py:96
msgid "HDFS url must not be empty"
msgstr ""

#: sahara/service/validations/edp/data_source.py:100
msgid "URL scheme must be 'hdfs'"
msgstr ""

#: sahara/service/validations/edp/data_source.py:102
msgid "HDFS url is incorrect, cannot determine a hostname"
msgstr ""

#: sahara/service/validations/edp/job.py:69
#, python-format
msgid "Job binary '%s' does not exist"
msgstr ""

#: sahara/service/validations/edp/job.py:83
#, python-format
msgid "%s job requires main application jar"
msgstr ""

#: sahara/service/validations/edp/job.py:86
#, python-format
msgid "%s flow requires main script"
msgstr ""

#: sahara/service/validations/edp/job.py:90
msgid "'mains' and 'libs' overlap"
msgstr ""

#: sahara/service/validations/edp/job.py:95
#, python-format
msgid "%s flow requires libs"
msgstr ""

#: sahara/service/validations/edp/job.py:98
#, python-format
msgid "%s flow does not use mains"
msgstr ""

#: sahara/service/validations/edp/job_binary_internal.py:27
#, python-format
msgid "%s is not a valid name"
msgstr ""

#: sahara/service/validations/edp/job_execution.py:58
#, python-format
msgid "%s job must specify edp.java.main_class"
msgstr ""

#: sahara/service/validations/edp/job_execution.py:74
#, python-format
msgid "%s job must specify streaming mapper and reducer"
msgstr ""

#: sahara/service/validations/edp/job_execution.py:83
#, python-format
msgid "Cluster with id '%s' doesn't exist"
msgstr ""

#: sahara/service/validations/edp/job_execution.py:91
#, python-format
msgid "Cluster with id '%(cluster_id)s' doesn't support job type '%(job_type)s'"
msgstr ""

#: sahara/service/validations/edp/job_execution.py:100
#, python-format
msgid "%s job requires 'input_id' and 'output_id'"
msgstr ""

#: sahara/topology/topology_helper.py:113
#, python-format
msgid "Unable to find file %s with compute topology"
msgstr ""

#: sahara/topology/topology_helper.py:129
#, python-format
msgid "Was not able to find compute node topology for VM %s"
msgstr ""

#: sahara/utils/api.py:137
msgid "Non-dict and non-empty kwargs passed to render"
msgstr ""

#: sahara/utils/api.py:159 sahara/utils/api.py:188
#, python-format
msgid "Content type '%s' isn't supported"
msgstr ""

#: sahara/utils/api.py:184
msgid "XML requests are not supported yet"
msgstr ""

#: sahara/utils/crypto.py:51
msgid "Private key file hasn't been created"
msgstr ""

#: sahara/utils/crypto.py:55
msgid "Public key file hasn't been created"
msgstr ""

#: sahara/utils/proxy.py:116
msgid "Proxy domain requested but not specified."
msgstr ""

#: sahara/utils/proxy.py:125
#, python-format
msgid "Failed to find domain %s"
msgstr ""

#: sahara/utils/proxy.py:129
#, python-format
msgid "Unexpected results found when searching for domain %s"
msgstr ""

#: sahara/utils/proxy.py:184
msgid "created proxy user {0}"
msgstr ""

#: sahara/utils/proxy.py:204
#, python-format
msgid "Failed to find user %s"
msgstr ""

#: sahara/utils/proxy.py:207
#, python-format
msgid "Unexpected results found when searching for user %s"
msgstr ""

#: sahara/utils/remote.py:134
msgid ""
"Remote driver is not loaded. Most probably you see this error because you"
" are running Sahara in distributed mode and it is broken.Try running "
"sahara-all instead."
msgstr ""

#: sahara/utils/ssh_remote.py:395
#, python-format
msgid "Closing HTTP session for %(host)s:%(port)s"
msgstr ""

#: sahara/utils/ssh_remote.py:401
#, python-format
msgid "Session for %(host)s:%(port)s not cached"
msgstr ""

#: sahara/utils/tempfiles.py:37
#, python-format
msgid "Failed to delete temp dir %(dir)s (reason: %(reason)s)"
msgstr ""

#: sahara/utils/types.py:88
#, python-format
msgid "Class %s is immutable!"
msgstr ""

#: sahara/utils/openstack/base.py:45
#, python-format
msgid "Endpoint with type %(type)s is not found for service %(service)s"
msgstr ""

#: sahara/utils/openstack/base.py:52
#, python-format
msgid "Service \"%s\" not found in service catalog"
msgstr ""

#: sahara/utils/openstack/heat.py:47
#, python-format
msgid "Failed to find stack %(stack)s"
msgstr ""

#: sahara/utils/openstack/neutron.py:78
#, python-format
msgid "Neutron router corresponding to network %s is not found"
msgstr ""

#: sahara/utils/openstack/neutron.py:185
#, python-format
msgid "Unknown file mode %s"
msgstr ""

